src/main/kotlin/xyz/atrius/waystones/commands/WarpstoneCommand.kt

```diff
@@ -49,23 +56,23 @@ object WarpstoneCommand : CommandExecutor {
         args: List<String>
     ): Boolean {
         // Set Amount and Player to give WarpKeys to
-        val amount = args.getOrNull(0)?.toIntOrNull() ?: 1
-        val player = args.getOrNull(1)?.let { Bukkit.getServer().getPlayer(it) } ?: sender as Player
+        fun getPlayerAndAmount(default: Player, first: String, second: String): Pair<Player, Int> {
+            val player = Bukkit.getPlayer(first) ?: default
+            val amount = second.toIntOrNull() ?: first.toIntOrNull() ?: 1
+            return player to amount
+        }
```

Comment:
While embedding functions is possible, I don't recommend doing things this way. Just place it in the top level of the class as a private function.